training-react/src/app/components/App/components/Field/index.js

```diff
@@ -1,12 +1,5 @@
 import React from 'react';
 
-export const customInput = props => {
-  const { label, input, meta, type } = props;
-  return (
-    <div>
-      <label htmlFor={label}>{label}</label>
-      <input {...input} type={type} id={label} />
-      {meta.error && meta.touched && <div style={{ color: 'red' }}>{meta.error}</div>}
-    </div>
-  );
-};
+import FieldDetails from './layout';
+
+export const customInput = props => <FieldDetails {...props} />;
```

Comment:
In this case you don't need the separation as this component doesn't have any logic.